### ISSUE TITLE
README.md Wifi Guide

### ISSUE THREAD

**Issue by Lewiscowles1986** [chriso0710/pikiosk#6]
from `README.md`

> When using Wifi configure Wifi settings via Ansible

if you create a valid `wpa_supplicant.conf` file and place in your `boot` partition the pi copies it over using stock rPi images (seems to now include lite)

the file disappears after it copies but activates network on first boot so you just have to ensure ssh service is enabled to use WiFi

**Comment by chriso0710** [comment]
Sounds great and I would like to add this to the README.
Did you test this successfully on a new Jessie image?

**Comment by Lewiscowles1986** [comment]
yes! Tested from Raspberry pi website for Jessie Lite & Pixel today :wink:

**Comment by chriso0710** [comment]
Added note in readme. 
Maybe we should link to an example for a valid wpa_supplicant.conf.

**Comment by Lewiscowles1986** [comment]
Would differ per-country because of radio laws... had a problem once because I forgot to set country in that file.
